tools/myst/enc/enc.c

```diff
@@ -521,6 +534,24 @@ static long _enter(void* arg_)
     /* Get the enclave size */
     enclave_size = __oe_get_enclave_size();
 
+    if (_get_config_id(config_id) != 0)
+    {
+        fprintf(stderr, "_get_config_id() failed\n");
+        assert(0);
+    }
+
+    myst_load_fssig(options->rootfs, &fssig);
+    if (_check_config_id(
+            config_id,
+            arg->config_data,
+            arg->config_size,
+            fssig.root_hash,
+            sizeof(fssig.root_hash)) != 0)
+    {
+        fprintf(stderr, "_check_config_id() failed\n");
+        assert(0);
+    }
+
```

Comment:
With configuration info passed in, the config_region is the enclave image is not needed anymore. It's OK to leave the enclave build process unchanged for now, for POC purpose, but the data in the config region should be ignored, and instead, the configuration data passed in should be used in the code below, after config_id based verification of the input.